.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
 /test.js
+/.vscode
```

Comment:
If you could configure your editor to add trailing empty lines that would be great!